packages/ckb-sdk-utils/src/serialization/script.ts

```diff
@@ -0,0 +1,21 @@
+import { serializeDynVec, serializeArray, serializeTable, serializeFixVec } from '.'
+
+export const serializeCodeHash = (codeHash: CKBComponents.Hash256) => serializeArray(codeHash)
```

Comment:
`serializeCodeHash` seems to be inner implementation. Perhaps should not be exported. (same for `serializeHashType`.

Reply:
![image](https://user-images.githubusercontent.com/7271329/64421535-605f1900-d0d4-11e9-8b46-d132d29bf0e0.png)

Only these methods exposed by the `utils` module, but users can still use the methods mentioned above by importing specific file (@nervosnetwork/ckb-sdk-utils/lib/serialization/*.js`. It's more flexible.